src/attestation/azext_attestation/generated/_params.py

```diff
@@ -44,3 +52,26 @@ def load_arguments(self, _):
         c.argument('resource_group_name', resource_group_name_type)
         c.argument('provider_name', options_list=['--name', '-n'], help='Name of the attestation service',
                    id_part='name')
+
+    for item in ['list', 'add', 'remove']:
+        with self.argument_context('attestation signer {}'.format(item)) as c:
+            c.extra('resource_group_name', resource_group_name_type, required=False)
+            c.extra('attestation_name', attestation_name_type, required=False)
```

Comment:
is name enough since command name is attestation already?

Reply:
does id work?

Reply:
> is name enough since command name is attestation already?

Yes, it's safe to shorten the parameter name. Good point.

Reply:
> does id work?

`Id` wasn't supported, it's better to add it.

Reply:
> is name enough since command name is attestation already?

The arg type `attestation_name_type` already has options list `--name -n`:
```python
attestation_name_type = CLIArgumentType(
        help='Name of the attestation.', options_list=['--name', '-n'], metavar='NAME', id_part=None,
        completer=get_resource_name_completion_list('Microsoft.Attestation/attestationProviders'))
```

Reply:
> does id work?

Support `--attestation-base-url/-u` to give users the ability to ignore `--name` and `--resource-group`:
```
    --attestation-base-url -u : URL of the attestation, for example:
                                https://myatt.eus2.attest.azure.net. You can ignore --name and
                                --resource-group if you specified the URL.
    --name -n                 : Name of the attestation.
    --resource-group -g       : Name of resource group. You can configure the default group using
                                `az configure --defaults group=<name>`.
```